proto/viam/app/data/v1/data.proto

```diff
@@ -564,3 +579,52 @@ message RemoveBinaryDataFromDatasetByIDsRequest {
 }
 
 message RemoveBinaryDataFromDatasetByIDsResponse {}
+
+message DataPipeline {
+  string id = 1;
+  string organization_id = 2;
+  string name = 3;
+  repeated bytes mql_binary = 4;
+  string schedule = 5;
+  string api_key_id = 6;
+  bool enabled = 7;
+}
```

Comment:
can you please copy in the field documentation for some of these? specifically `mql_binary` `schedule` and `api_key_id` might not be trivial